wolox-books/src/scss/colors.scss

```diff
@@ -0,0 +1,6 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$lightgrey: #f1f1f1;
+$white: #ffffff;
```

Comment:
if the color has the shape #AABBCC you can reduce it to #ABC, for this case it would be #FFF